src/ResourceManager/DeviceProvisioningServices/DeviceProvisioningServices.sln

```diff
@@ -0,0 +1,94 @@
+﻿
```

Comment:
@anusapan you will need to add the following projects to your solution:
- `Commands.ServiceManagement.Common` (`"..\..\ServiceManagement\Common\Commands.ServiceManagement.Common\Commands.ServiceManagement.Common.csproj"`)
- `Commands.Common.Authentication.ResourceManager` (`..\Common\Commands.Common.Authentication.ResourceManager\Commands.Common.Authentication.ResourceManager.csproj`)
- `Commands.Common.Compute` (`..\..\Common\Commands.Common.Compute\Commands.Common.Compute.csproj`)

Reply:
Added